### PR TITLE
Add support for Access Logs in BackendConfig

### PR DESCRIPTION
**Pull request by skmatti**

### CODE REVIEW

**Comment by freehan** [comment]
Is this still needed?

**Comment by skmatti** [comment]
Yes, this will go into v1.9.0. Older versions do not need this as a feature. But, starting v1.9.0, users can configure access logs as they like.

File: pkg/backends/features/logging.go [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+}
+
+// expectedBackendServiceLogConfig returns composite.BackendServiceLogConfig for
+// the access log settings specified in the BackendConfig to the passed in
+// composite.BackendService.
+func expectedBackendServiceLogConfig(sp utils.ServicePort) *composite.BackendServiceLogConfig {
+	logConfig := &composite.BackendServiceLogConfig{
+		Enable: sp.BackendConfig.Spec.Logging.Enable,
+	}
+	if !sp.BackendConfig.Spec.Logging.Enable || sp.BackendConfig.Spec.Logging.SampleRate == nil {
```
Comment (freehan) [inline pkg/backends/features/logging.go]: you need to check if Logging is nil?

Comment (skmatti) [reply pkg/backends/features/logging.go]: This is checked in caller function. Added a comment that this is assumed to be not nil. Would it be cleaner if I explicitly include a check here as well?.

**Pull request merged**